sdk/identity/azure-identity/azure/identity/_credentials/azure_cli.py

```diff
@@ -36,12 +36,13 @@ class AzureCliCredential(object):
 
     This requires previously logging in to Azure via "az login", and will use the CLI's currently logged in identity.
 
-    :keyword str tenant_id: optional tenant to include in the token request.
+    :keyword List[str] additionally_allowed_tenants: optional additional tenant ids for which the credential
```

Comment:
Same (nitpicking) comment from earlier about capitalizing the O in "optional" -- this applies to all of the docstrings with `additionally_allowed_tenants` 🙂

Reply:
Checked other libraries and I believe we should captalize it.